nomos-services/blend/src/lib.rs

```diff
@@ -250,9 +250,15 @@ pub struct BlendConfig<BackendSettings, BackendNodeId> {
         Vec<Node<BackendNodeId, <SphinxMessage as nomos_blend_message::BlendMessage>::PublicKey>>,
 }
 
+#[cfg_attr(feature = "time", serde_with::serde_as)]
 #[derive(Serialize, Deserialize, Clone, Debug)]
 pub struct CoverTrafficExtSettings {
+    #[cfg_attr(
+        feature = "time",
+        serde_as(as = "MinimalBoundedDuration<432000, SECOND>")
+    )]
     pub epoch_duration: Duration,
+    #[cfg_attr(feature = "time", serde_as(as = "MinimalBoundedDuration<1, SECOND>"))]
     pub slot_duration: Duration,
 }
```

Comment:
Could we set the `epoch_duration` to 1s as well for now? We'll highly likely set this value to 432000, but strictly speaking, this value should be the same as the epoch duration that the consensus service uses. That value is calculated by the [epoch_length](https://github.com/logos-co/nomos/blob/b136f6466e51585fd6a78acc2ecad4b0b4b2aae7/consensus/cryptarchia-engine/src/time.rs#L112C12-L112C24) function. I'll figure out how to simply this `CoverTrafficExtSettings` by obtaining values from the consensus config. Until then, I think that the manual setting is fine, and 1s is a safe minimum.

Reply:
I just discussed with Marcin. The conclusion is that these parameters are independent with the ones in the consensus service. So, it's reasonable to have these here indepdently.